src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -145,7 +142,7 @@ private UUID printApplicantConsentOrderApprovedDocuments(CaseDetails caseDetails
         return bulkPrintFinancialRemedyLetterPack(caseDetails.getId(), applicantDocuments);
     }
 
-    private UUID bulkPrintFinancialRemedyLetterPack(Long caseId, List<BulkPrintDocument> documents) {
+    public UUID bulkPrintFinancialRemedyLetterPack(Long caseId, List<BulkPrintDocument> documents) {
```

Comment:
if you're making this public make sure you've added unit tests

Reply:
good spot - this was a hangover from previous refactoring no longer needed - reverted this change.